fe/be-java-extensions/max-compute-scanner/src/main/java/org/apache/doris/maxcompute/MaxComputeColumnValue.java

```diff
@@ -83,8 +84,8 @@ private void skippedIfNull() {
     @Override
     public boolean getBoolean() {
         skippedIfNull();
-        TinyIntVector tinyIntCol = (TinyIntVector) column;
-        return tinyIntCol.get(idx++) > 0;
+        BitVector bitCol = (BitVector) column;
```

Comment:
Please explain why using bitvector here?
and what about tinyint?

Reply:
bool type is bind to bit vector
tinyint type  bind to tinyint vector, it can work